styles/themes/charcoal.css

```diff
@@ -1790,6 +1790,9 @@ hr.divider {
   text-align: center;
   text-shadow: 3px 2px 5px black;
 }
+.rss-icon {
+  color: #f26522;
+}
```

Comment:
After looking at this actual size on Charcoal, I like it less than I initially thought I would (it looks fine on the white-based themes) -- I've done a bit of research, and it looks like there is no one true color for the RSS icon as far as the internet is concerned, so, if I may suggest an alternative for Charcoal ... I tried #ff9a00 (another, somewhat lighter version), and then tried mixing the two. #fd9205 looks good to me, and is slightly darker but not as vibrant against the very dark grey background as the existing color.

IIRC, it means adding an override to `charcoal.less`. I don't feel like it has to happen in this PR, but wanted to mention it here first.

Reply:
I tried it with #fd9205, I have no preference of one color over the other. It is an easy change, so I'm happy to add this color for the charcoal theme in this PR if that's what the team prefers.

Reply:
Screenshots for everyone to look at:

Current sandbox (#f26522), charcoal: 
<img width="149" alt="image" src="https://github.com/DistributedProofreaders/dproofreaders/assets/36800239/3976b734-7285-4415-b00a-9ff13b99a50a">

Charcoal with #fd9205: 
<img width="144" alt="image" src="https://github.com/DistributedProofreaders/dproofreaders/assets/36800239/f47a03da-aaab-4fdb-a773-cc5a641fceb0">

default: 
<img width="146" alt="image" src="https://github.com/DistributedProofreaders/dproofreaders/assets/36800239/7f5ab07e-07d1-46ec-91be-4fc113333653">

default with #fd9205: 
<img width="142" alt="image" src="https://github.com/DistributedProofreaders/dproofreaders/assets/36800239/20ad4e61-4b7c-43b7-bc31-51ffe34024b2">

(Wow ... I had no idea that the background for charcoal was so close to the background (in dark mode) for github 😁 )

Reply:
I think the current color on white looks good and defer to @srjfoo on what color she thinks is best for charcoal. If we can get it into this PR that would be appreciated.

Reply:
I updated the PR to use #fd9205 for charcoal.